dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/staxutils/StaxUtils.java

```diff
@@ -1952,9 +1980,13 @@ public static void printXmlFragment(XMLStreamReader reader) {
                 StaxUtils.close(writer);
             }
 			// Liberty Change - Prevent from going to message logs
-            LOG.finest(sw.toString());
+            if (LOG.isLoggable(Level.FINEST)) { // Liberty Change
+                LOG.finest(sw.toString());
+            }
         } catch (XMLStreamException e) {
-            LOG.severe(e.getMessage());
+            if (LOG.isLoggable(Level.FINEST)) { // Liberty Change
```

Comment:
I don't think you want to have these check?  It is a change in behavior to no longer log a severe here unless trace is enabled.

Reply:
Yep, you're right. Must have been blindly adding it.